roles/network_plugin/calico/templates/calico-cr.yml.j2

```diff
@@ -39,8 +42,6 @@ rules:
     verbs:
     - use
 {% elif calico_datastore == "kdd" %}
-      # Calico stores some configuration information in node annotations.
-      - update
```

Comment:
Why do we need to move this part for eBPF?

Reply:
This is actually an unrelated fix, if you look at the upstream manifests the `update` applies to the `nodes/status` resource not `policy`. I can't help feeling this was actually a security issue in kubespray deployments allowing Calico service account to update policies that it shouldn't.